### PR TITLE
Fix `default` -> `auto` prior parameter in documentation for lda-related models

### PR DESCRIPTION
**Pull request by Laubeee**
Change "default" to "auto" since there is no handling on the value "default".

### CODE REVIEW

File: gensim/models/ldamodel.py [hunk 1]
```diff
@@ -369,7 +369,7 @@ def __init__(self, corpus=None, num_topics=100, id2word=None,
             Alternatively default prior selecting strategies can be employed by supplying a string:
 
                 * 'asymmetric': Uses a fixed normalized asymmetric prior of `1.0 / topicno`.
-                * 'default': Learns an asymmetric prior from the corpus.
+                * 'auto': Learns an asymmetric prior from the corpus.
```
Comment (menshikh-iv) [inline gensim/models/ldamodel.py]: please update this in other places too

https://github.com/RaRe-Technologies/gensim/blob/56070fbb772259d434af957f46b4cda8954f4927/gensim/models/ldamodel.py#L534
https://github.com/RaRe-Technologies/gensim/blob/56070fbb772259d434af957f46b4cda8954f4927/gensim/models/ldamulticore.py#L132
https://github.com/RaRe-Technologies/gensim/blob/466b32fe6b5631403fe5be8c16233df3d61a373b/gensim/sklearn_api/ldamodel.py#L64
https://github.com/RaRe-Technologies/gensim/blob/466b32fe6b5631403fe5be8c16233df3d61a373b/gensim/sklearn_api/atmodel.py#L86

**Comment by Laubeee** [comment]
Oh alright didn't catch those
Were these all the places?

File: gensim/sklearn_api/ldamodel.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def __init__(self, num_topics=100, id2word=None, chunksize=2000, passes=1, updat
             Alternatively default prior selecting strategies can be employed by supplying a string:
 
                 * 'asymmetric': Uses a fixed normalized assymetric prior of `1.0 / topicno`.
-                * 'default': Learns an assymetric prior from the corpus.
+                * 'auto': Learns an assymetric prior from the corpus.
```
Comment (piskvorky) [inline gensim/sklearn_api/ldamodel.py]: `assymetric` => `asymmetric` (here and elsewhere)

**Comment by menshikh-iv** [comment]
Congratz with the first contribution @Laubeee :+1:

**Pull request merged**